src/OrchardCore/OrchardCore.Contents.Core/ContentsAdminListFilterOptions.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+
+namespace OrchardCore.Contents;
+
+public class ContentsAdminListFilterOptions
+{
+    public const string DefaultTermName = "text";
+
+    private readonly Dictionary<string, string> _maps = new(StringComparer.OrdinalIgnoreCase);
+
+    /// <summary>
+    /// Gets the term-name to use by the given content type.
+    /// </summary>
+    /// <param name="contentType"></param>
+    /// <param name="termName"></param>
+    /// <returns></returns>
+    public bool TryGetValue(string contentType, out string termName)
+    {
+        return _maps.TryGetValue(contentType, out termName);
```

Comment:
Maybe `TryGetValue()` => `TryGetDefaultTermNameByContentType()` or if too long `TryGetDefaultTermName()`

In case we may have other options in the future.

Hmm, then maybe `_maps` => `_defaultTermNamesByContentType` or just `_defaultTermNames`